ibis/pandas/client.py

```diff
@@ -29,6 +29,7 @@
     'string': 'string',
     'unicode': 'string',
     'bytes': 'string',
+    'empty': 'string',
```

Comment:
I think this should actually be `'null'` since ibis has a notion of a `null` type.

Reply:
oh, let me see

Reply:
yep its ok